src/ngraph/builder/quantization.cpp

```diff
@@ -121,23 +121,50 @@ namespace ngraph
                                                                      max_input,
                                                                      min_filter,
                                                                      max_filter,
                                                                      min_freezed_output,
-                                                                     max_freezed_output);
+                                                                     max_freezed_output,
+                                                                     output_et);
 
-            return make_shared<op::QuantizedConvolutionBias>(input,
-                                                             filters,
-                                                             bias,
-                                                             window_movement_strides,
-                                                             window_dilation_strides,
-                                                             padding_below,
-                                                             padding_above,
-                                                             data_dilation_strides,
-                                                             requantization_scale,
-                                                             with_relu);
+            if (bias->get_element_type() != element::i32)
+            {
+                auto zero = make_constant(element::i32, min_input->get_shape(), 0);
+                AxisSet quantization_axes;
+                auto bias_scale =
+                    quantization_util::get_bias_scale(min_input, max_input, min_filter, max_filter);
+                op::Quantize::RoundMode round_mode =
+                    op::Quantize::RoundMode::ROUND_NEAREST_TOWARD_EVEN;
+
+                auto reordered_bias = make_shared<op::Quantize>(
+                    bias, bias_scale, zero, element::i32, quantization_axes, round_mode);
+                return make_shared<op::QuantizedConvolutionBias>(input,
+                                                                 filters,
+                                                                 reordered_bias,
+                                                                 window_movement_strides,
+                                                                 window_dilation_strides,
+                                                                 padding_below,
+                                                                 padding_above,
+                                                                 data_dilation_strides,
+                                                                 requantization_scale,
+                                                                 with_relu);
+            }
+            else
+            {
+                return make_shared<op::QuantizedConvolutionBias>(input,
```

Comment:
There's no need to have multiple return statements here.  Instead, just set bias = new bias.